crates/astria-composer/src/executor/bundle_factory/mod.rs

```diff
@@ -87,16 +87,21 @@ impl SizedBundle {
             return Err(SizedBundleError::SequenceActionTooLarge(seq_action));
         }
 
-        if self.curr_size + seq_action_size > self.max_size {
+        let new_size = self
+            .curr_size
+            .checked_add(seq_action_size)
```

Comment:
IMO this one extra test here is redundant and we can just do a `saturating_add` here: if `new_size = u64::MAX` and `self.max_size = u64::MAX`, then `new_size > self.max_size == false` always.

Reply:
Good point!  Done in [44c99af](https://github.com/astriaorg/astria/pull/1081/commits/44c99affc52d66ee3134fa1bbc23f66d21bb632a).